deps.edn

```diff
@@ -401,6 +401,73 @@
 
 ;;; Linters
 
+  ;; clojure -M:kondo --lint src test
+  ;;
+  ;; clojure -M:kondo --version
+  ;;
+  ;; clojure -M:kondo --copy-configs --dependencies --lint "$(clojure -A:dev -Spath)" --skip-lint --parallel
+  ;;
+  ;; Run Kondo from the JVM using the pinned version. Preferable to running the installed command since we can pin the
+  ;; version here which may be different from the version installed on your computer.
+  ;;
+  ;; Use this to only run Kondo against specific files.
+  :kondo
+  {:replace-deps
+   {clj-kondo/clj-kondo {:mvn/version "2024.08.01"}}
```

Comment:
We are unfortunately specifying the Kondo dep twice now (here and in `:dev` as well). I was going to ask people to run

```
clojure -M:dev:kondo
```

but that ends up grabbing all of the deps for both normal mode and dev and adding them to the classpath when we didn't really need any of them to run Kondo -- that extra overhead adds a few seconds, which may not sound like much but adds up for a linter. By adding a separate dep here I was able to use `:replace-deps` and avoid all the other unneeded stuff.